hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -374,12 +374,9 @@
           name: "FIPS_ENABLE",
           mubi: true,
           desc: '''
-                Setting this field to `kMultiBitBool4True` selects the FIPS/CC compliant mode (or short FIPS mode).
-                In this mode, the ENTROPY_SRC block can generate FIPS qualified entropy.
-                Additional requirements to generate FIPS qualified entropy are i) that at most one of the !!ENTROPY_CONTROL.ES_ROUTE and !!ENTROPY_CONTROL.ES_TYPE fields are set to `kMultiBitBool4True` but not both, and ii) that !!CONF.RNG_BIT_ENABLE is set to `kMultiBitBool4False`.
-
                 Setting this field to `kMultiBitBool4False` selects the boot-time / bypass mode in which the hardware conditioning is bypassed.
-
+                The conditioner will only be used if exactly one of !!ENTROPY_CONTROL.ES_ROUTE and !!ENTROPY_CONTROL.ES_TYPE is set to `kMultiBitBool4True`.
```

Comment:
With the new text, it is not entire obvious why this field is called `FIPS_ENABLE`, especially that now there is a `FIPS_FLAG` field.

Reply:
Thanks for reviewing @pamaury. It's an artifact of the previous implementation for the FIPS bit. It would probably be better if it was renamed to something like bypass_enable but this would involve way too much effort for little benefit. I'll update this comment to make this clear.

Reply:
Good point @pamaury . @h-filali and I discussed this in person now and we think some more changes are required.

The `FIPS_ENABLE` field controls several things:
1. Which health test window sizes and thresholds etc. are used (there are two registers for FIPS mode and non-FIPS mode).
2. Whether the block runs in continuous or boot-time mode. The latter just generates one single seed. After that, the ENTROPY_SRC block needs to be restarted.
3. It enables the conditioner unless other settings see `ES_TYPE` and `ES_ROUTE` are set accordingly. If these are set, no entropy can flow to CSRNG. So this means, whenever the ENTROPY_SRC block can forward entropy to CSRNG and any consuming endoints, the conditioner is on.

A closer investigation of the main FSM reveales that if `FIPS_ENABLE == false` and enabling Firmware Override: Extrand & Insert mode (i.e. our holy software fallback mode), more than one seed can be generated WITHOUT hardware conditioning. So this should be fine but we need for sure to document this.

Reply:
@h-filali , please replace this by something like:
```
Setting this field to `kMultiBitBool4True` selects the mode targeting FIPS/CC compliance (in short FIPS mode) with hardware conditioning enabled.
The ENTROPY_SRC block will use the FIPS_WINDOW, FIPS_THRESH and FIPS_WATERMARK values of the !!HEALTH_TEST_WINDOWS, health test thresholds and watermark register, respectively. 
Whether the ENTROPY_SRC block is indeed running with the configuration used for FIPS/CC validation is under the control of firmware.
Thus, firmware must explicitly mark the produced entropy as FIPS qualified using the !!CONF.FIPS_FLAG field.
Note that the hardware conditioning can still be disabled in FIPS mode by setting both !!ENTROPY_CONTROL.ES_ROUTE and !!ENTROPY_CONTROL.ES_TYPE fields to `kMultiBitBool4True`.
However, no entropy is being passed to the block hardware interface in this mode.

Setting this field to `kMultiBitBool4False` selects the boot-time / bypass mode in which the hardware conditioning is bypassed.
```

Reply:
How to use the Firmware Override: Extract and Insert mode WITHOUT conditioning should probably be explicitly stated in the programmers guide [here](https://opentitan.org/book/hw/ip/entropy_src/doc/programmers_guide.html#firmware-override---extract--insert). This can be done in a follow-up PR though.

Reply:
I'll replace the text with yours and make a follow up PR with the programmers guide change.